molgenis-security/src/main/java/org/molgenis/security/MolgenisWebAppSecurityConfig.java

```diff
@@ -139,7 +142,7 @@ protected void configure(HttpSecurity http) throws Exception
 
 		.formLogin().loginPage("/login").failureUrl("/login?error").and()
 
-		.logout().addLogoutHandler((req, res, auth) -> {
+		.logout().deleteCookies("JSESSIONID").addLogoutHandler((req, res, auth) -> {
```

Comment:
All containers must use JSESSIONID.
Needed because when you sign out your sessionid is invalid, so when you go to the molgenis login form it will shoe the message 'Your login session is expired', that is a bit odd.